content/modules/modules-quickstart.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Modules Quick Start
+description:
```

Comment:
@bmansheim this does not contain a link to where they need to sign up.  We should have this no?
Also this is not the only quick start for modules, there are other ways to do it.  I don't think this is clear out of this document